### PR TITLE
Sus 181/adding support endpoints

### PR DESCRIPTION
**Pull request by JiaanNie**

### CODE REVIEW

File: src/services/sam/supports/ticketTypes.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    subject: string,
+    priority: string,
+  }
+}
+export interface createContactRequest {
+  email: string,
+  first_name: string,
+  last_name: string,
+  phone_number: string
+
```
Comment (ghost) [inline src/services/sam/supports/ticketTypes.ts:21]: So minor but there's an extra space here

File: src/services/sam/supports/tickets.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    this.axiosClientHeaderless = makeAxiosClientHeaderless(axiosClient);
+    this.logger = logger;
+  }
+  create_ticket(payload: createTicketRequest, projectUuid: string, activeMemberId: string) {
+    this.axiosClient.post(`/sam/support/project/${projectUuid}/members/${activeMemberId}/tickets`, payload).then((res: any) => {
+      this.logger.debug(res);
+    });
+  }
+  list_tickets(projectUuid: string,  activeMemberId: string) {
+
```
Comment (ghost) [inline src/services/sam/supports/tickets.ts:31]: Some random spaces in here

**Comment by ShawnClake** [comment]
Just run a ctrl + alt + L to auto format your files :) then @jherasymuik-ehelply wont complain about spacing :D

**Review by ShawnClake — APPROVED** [review]

File: src/services/sam/supports/tickets.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  axiosClient: AxiosInstance
+  axiosClientHeaderless: AxiosInstance
+  logger: Logger
+  constructor(axiosClient: any, logger: Logger) {
+    this.axiosClient = axiosClient;
+    this.axiosClientHeaderless = makeAxiosClientHeaderless(axiosClient);
+    this.logger = logger;
+  }
+  create_ticket(payload: createTicketRequest, projectUuid: string, activeMemberId: string) {
+    this.axiosClient.post(`/sam/support/project/${projectUuid}/members/${activeMemberId}/tickets`, payload).then((res: any) => {
```
Comment (ghost) [inline src/services/sam/supports/tickets.ts]: Do we want these to return something?

Comment (ShawnClake) [reply src/services/sam/supports/tickets.ts]: wait a second.... yes all of these should be returning the promise

**Review by ShawnClake — CHANGES_REQUESTED** [review]
Each sdk function needs to return the promise

**Pull request merged**